implementations/micrometer-registry-datadog/src/main/java/io/micrometer/datadog/DatadogConfig.java

```diff
@@ -72,10 +80,73 @@ default boolean descriptions() {
         return getBoolean(this, "descriptions").orElse(true);
     }
 
+    /**
+     * @return {@code true} if client-side telemetry should be enabled when in Dogstatsd
+     * mode.
+     * https://docs.datadoghq.com/developers/dogstatsd/high_throughput/?code-lang=java#client-side-telemetry
+     */
+    default boolean enableClientSideTelemetry() {
+        return getBoolean(this, "disableClientSideTelemetry").orElse(true);
+    }
+
+    /**
+     * @return The maximum packet size to buffer before sending metrics when using
+     * Dogstatsd mode.
+     * https://docs.datadoghq.com/developers/dogstatsd/high_throughput/?code-lang=java#enable-buffering-on-your-client
+     */
+    default Integer maxPacketSizeBytes() {
+        return getInteger(this, "maxPacketSizeBytes").orElse(-1);
+    }
+
+    /**
+     * @return {@code false} if client-side aggregation should be enabled when in
+     * Dogstatsd mode.
+     * https://docs.datadoghq.com/developers/dogstatsd/high_throughput/?code-lang=java#client-side-aggregation
+     */
+    default boolean enableAggregation() {
+        return getBoolean(this, "enableAggregation").orElse(false);
+    }
+
+    /**
+     * @return {@code 3000} sets the period of time in milliseconds in which the
+     * aggregator will flush its metrics into the sender when in Dogstatsd mode.
+     * https://github.com/DataDog/java-dogstatsd-client#configuration-1
+     */
+    default Integer aggregationFlushInterval() {
+        return getInteger(this, "aggregationFlushInterval").orElse(3000);
+    }
+
+    /**
+     * @return {@code 4} determines the number of shards in the aggregator, this feature
+     * is aimed at mitigating the effects of map locking in highly concurrent scenarios
+     * when in Dogstatsd mode.
+     * https://github.com/DataDog/java-dogstatsd-client#configuration-1
+     */
+    default Integer aggregationShards() {
+        return getInteger(this, "aggregationShards").orElse(4);
```

Comment:
I wonder if it makes sense to expose this given the setup of Micrometer doing the aggregating and the DogStatsD client sending metrics once per interval. With that, I doubt anyone would need to configure this.